components/tidb_query/src/batch/executors/top_n_executor.rs

```diff
@@ -46,6 +47,7 @@ pub struct BatchTopNExecutor<Src: BatchExecutor> {
     eval_columns_buffer_unsafe: Box<Vec<RpnStackNode<'static>>>,
 
     order_exprs: Box<[RpnExpression]>,
+    order_exprs_field_type: Box<[FieldType]>,
```

Comment:
Could we add some comments to this field? Since the fields above it are quite documented.

Reply:
done